### PR TITLE
Fxfs

### PR DESCRIPTION
**Pull request by mehdibalouchi**
Add a new JWE field to the file.

### CODE REVIEW

File: importer/balanced/builder.go [hunk 1]
```diff
@@ -262,3 +265,108 @@ func fillNodeRec(db *h.DagBuilderHelper, node *h.FSNodeOverDag, depth int) (fill
 
 	return filledNode, nodeFileSize, nil
 }
+
+func LayoutEnc(db *h.DagBuilderHelper, jwe []byte) (ipld.Node, error) {
+	fmt.Println("inside LayoutEnc rrrrrrrrrrrrrrrrrrr", jwe)
+	if db.Done() {
+		fmt.Println("inside LayoutEnc rrrrrrrrrrrrrrrrrrrdddd", jwe)
```
Comment (masih) [inline importer/balanced/builder.go]: Consider using https://github.com/ipfs/go-log

Please have a search for `fmt.Println` across the PR and replace as needed.

Comment (mehdibalouchi) [reply importer/balanced/builder.go]: done

File: importer/helpers/dagbuilder.go [hunk 2]
```diff
@@ -172,6 +172,38 @@ func (db *DagBuilderHelper) NewLeafNode(data []byte, fsNodeType pb.Data_DataType
...
+		if err != nil {
+			return nil, err
+		}
+		return rawnode, nil
+	}
+
+	// Encapsulate the data in UnixFS node (instead of a raw node).
+	fsNodeOverDag := db.NewEncFSNodeOverDag(fsNodeType, jwe)
+	fsNodeOverDag.SetFileData(data)
+	// node, err := fsNodeOverDag.Commit()
```
Comment (masih) [inline importer/helpers/dagbuilder.go:195]: Do we still need this line?

**Review by masih — APPROVED** [review]
I'd love to see some tests for the added functionality. Please feel free to add the tests in a separate PR

**Pull request merged**